### ISSUE TITLE
Expose Pod Information to Containers Through Environment Variables 

### ISSUE THREAD

**Issue by xhensiladoda** [raffis/mongodb-query-exporter#54]
#### Is your feature request related to a problem? Please describe
I need to [expose pod information](https://kubernetes.io/docs/tasks/inject-data-application/environment-variable-expose-pod-information/) on my custom metrics from the container.
In more specific, I need to add the following environment variable:
```
env:
    - name: POD_NAME
      valueFrom:
          fieldRef:
              fieldPath: metadata.name
```
Currently, this is not supported on the [deployment](https://github.com/raffis/mongodb-query-exporter/blob/master/chart/prometheus-mongodb-query-exporter/templates/deployment.yaml#L37-L55).
#### Describe the solution you'd like
Ideally, I would like to have a `key/value` filed where:
-  `key` is the environment variable name
- `value` is the `fieldPath` where to retrieve the pod information

Example:
```
envFromFieldPath:
    POD_NAME: "metadata.name"
```

#### Describe alternatives you've considered
I have considered changing myself the chart, and adding in the _deployment.yaml_:
```
        {{- range $key, $value := .Values.extraEnvFieldPath }}
          - name: {{ $key }}
            valueFrom:
              fieldRef:
                fieldPath: {{ $value }}
          {{- end }}
```
Then, I specify in my _values.yaml_:
```
  extraEnvFieldPath:
    POD_NAME: "metadata.name"
```
After I install the chart, I get on the container environment variables:
```
    Environment:
      POD_NAME:                 metrics-exporter-prometheus-mongodb-query-exporter-6545c56fbxgv (v1:metadata.name)
```

#### Additional context
NA

**Comment by raffis** [comment]
@[xhensiladoda](https://github.com/xhensiladoda) legit request, feel free to open a pr with your proposed change.

**Comment by raffis** [comment]
fixed by #55
